src/runtime/meter-toast-api.js

```diff
@@ -160,6 +170,26 @@ export class MeterToastApi {
     });
   }
 
+  /**
+   * Changes the size of the loading iframe on desktop to match the size of
+   * the meter toast iframe.
+   */
+  setLoadingViewWidth_() {
+    const mq = this.win_.matchMedia(
+      '(min-width: 640px) and (min-height: 640px)'
+    );
+    const element = this.dialogManager_
+      .getDialog()
+      .getLoadingView()
+      .getElement();
+    if (mq.matches) {
+      setImportantStyles(element, {
```

Comment:
Could you cover this line with tests?

Reply:
I don't know how to do that :( I haven't seen any testing patterns to follow for UI testing before and I am unsure how to implement it. Do you have examples or advice? The closest I've found is [dialog-test.js](https://github.com/subscriptions-project/swg-js/blob/main/src/components/dialog-test.js).

Reply:
I think you could extract the desktop media query to a new method, and then override it in a test to return `true`

Reply:
The `getStyle` method used in dialog-test.js could work well for verifying the styles

Reply:
Thanks for the advice! Added tests/